app/routes/verkiezingen.js

```diff
@@ -0,0 +1,30 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
```

Comment:
remove `inject as`

Reply:
feels like a thing we do in general and should fix in one pass. won't be stopping this pr for it